### DISCUSSION TITLE
RESEARCH: Lightning over Ham-Radio

### DISCUSSION THREAD

**Discussion by rootzoll** [raspiblitz/raspiblitz#3907] [Research]
Take a look at this c-lightning experiment ... would be great to have a similar option for RaspiBlitz:
https://twitter.com/remyers_/status/1295069320482033664?s=20
https://inthemesh.com/archive/fldigi-proxy/

**Comment by woeisme** [comment]
There's also data over hamband called [APRS](https://en.wikipedia.org/wiki/Automatic_Packet_Reporting_System) they even have repeaters in the ISS.

**Comment by rootzoll** [comment]
More Background info APRS: https://www.kj7nzl.net/blog/sending-sms-messages-through-iss/
